PlanMyTrip/PlanMyTrip/View Controllers/DisplayResultsViewController.m

```diff
@@ -119,11 +143,12 @@ - (void)viewDidLoad {
                 NSString *rating = [item objectForKey:@"rating"];
                 
                 if(count == 1){
-                    NSString *test = [NSString stringWithFormat:@"The first hotel recomendation is the %@. It costs %@ per night and is rated %@ stars.", name, price, rating];
```

Comment:
each of these block looks almost identical. we should avoid code duplication as much as possible. For example, you don't need count == 1 to map to string "first hotel". Simply format 1 into the sentence is the same.